cluster-bundle.js

```diff
@@ -121,44 +122,28 @@ function writeBundleMapToDisk(bundle) {
     });
 }
 
-function writeConfig() {
-    var bundles = Object.keys(processedBundles).map(function (key) {
-        return processedBundles[key];
-    });
-
-    var configFilePath = path.join(jspmBaseUrl, 'systemjs-bundle-config.js');
-    var configFileData = 'System.config({ bundles: ' + JSON.stringify(bundles, null, '\t') + ' })';
-
-    console.log('writing to %s', configFilePath);
-    fs.writeFile(configFilePath, configFileData, function (e) {
-        if (e) return process.exit(1);
-        process.exit(0);
-    });
-}
-
 function updateBundles(message) {
     processedBundles[message.id] = message.configs;
 }
 
 function writeConfig() {
-    var configs = Object.keys(processedBundles).reduce(function (acc, key) {
-        var bundles = processedBundles[key];
-
-        Object.keys(bundles).forEach(function (bundleKey) {
-            acc[bundleKey] = bundles[bundleKey];
-        });
-
-        return acc;
-    }, {});
+    var configs = _(processedBundles)
+        .map()
```

Comment:
using `map` without passing it a mapping function doesn't make sense at all. 

the only reason this code works is because lodash passes `_.identity` to map by default if you don't provide your own function. anyone not familiar with that behaviour (which is unique to lodash) would need to look up the documentation to understand this code - i don't think that counts as a readability improvement